physical_properties/pKa/analysis/analysis_of_typeIII_predictions/typeIII_analysis.py

```diff
@@ -66,6 +66,8 @@ def compute_bootstrap_statistics(samples, stats_funcs, percentile=0.95, n_bootst
         stats_funcs = [stats_funcs]
 
     # Compute mean statistics.
+    #print("\nSamples:\n", samples)
+
```

Comment:
@MehtapIsik Is there a reason to have these print statements in the code, but commented out. I would say you should either include them or delete the lines.

Reply:
I'll do a little bit of cleaning.